doc/rest.md

```diff
@@ -416,6 +416,10 @@ Query parameters:
   - `full_text` Publications will be ordered by results of full-text search. Can be used only in combination with *full_text_filter*.
   - `title` Publications will be ordered lexicographically by title value.
   - `last_change` Publications will be ordered by time of last change. Recently updated publications will be first.
+  - `bbox` Publications will be ordered by similarity of bounding box with bounding box passed in *ordering_bbox* or *bbox_filter*. Can be used only in combination with  *ordering_bbox* or *bbox_filter*.
```

Comment:
Remove `By default, maps are ordered by search rank in response if this filter is used.` at `full_text_filter`.